src/config/Config.ts

```diff
@@ -526,7 +529,7 @@ export class BridgeConfig {
     @hideKey()
     private readonly bridgePermissions: BridgePermissions;
 
-    constructor(configData: BridgeConfigRoot, env?: {[key: string]: string|undefined}) {
+    constructor(configData: BridgeConfigRoot, env?: { [key: string]: string | undefined }) {
         this.bridge = configData.bridge;
         assert.ok(this.bridge);
         this.github = configData.github && new BridgeConfigGitHub(configData.github);
```

Comment:
Would suggest modifying line 552 so that if `enabled` is false, we treat it as `{ monolithic: true}`

Reply:
Handled in latest commit - I'm not a great JS dev, feel free to critique